### PR TITLE
Update meetings page with bylaws and remove standups

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

**Review by patcon — APPROVED** [review]
LGTM!

**Comment by patcon** [comment]
Thanks dc!

Seeing a couple non-blocking link failures. Re-ran CI and still getting them:
- https://cloud.hypha.coop Asked infra about it in chat
- https://link.hypha.coop/twitter might be because we now do those redirects in JS, and our link-checker isn't smart

I think this is out of scope for this PR, but any objection to my mentioning how "WG meetings" work (or whatever we want to call them, can decide in PR). The page rn kinda over-represents the sorts of meetings that make up the least of the day-to-day experience of being in Hypha :)

File: meetings.md [hunk 1]
```diff
@@ -22,31 +23,37 @@ All meetings are expected to have someone in the **Cultivator** &
...
-📋 **Template:** [`link.hypha.coop/standup-template`](https://link.hypha.coop/standup-template)
+#### Annual Members Meeting
+
+📆 **Schedule:** Annually, by our [Bylaws](https://handbook.hypha.coop/bylaws.html#3-member-meetings) they "shall be held within 4 months of the fiscal year end and within 15 months of any previous annual meetings", with appropriate notice given
+🕙 **Duration:** 0.5 day 
+📋 **Template:** 
+
+Members meetings are our annual review where we elect the board, accept finance statements, and hear updates on the work done over the previous year. We tend to schedule this with a board meeting and strategic retreat.
+
+##### Special Members Meeting
```
Comment (patcon) [inline meetings.md:46]: Is the hierarchy of All Hands > Annual > Special what is intended?

Comment (dcwalk) [reply meetings.md:46]: oop, know I meant it as in the TOC! I'll fix
  - [All Hands](#all-hands)
  - [All Hands: Standup](#standup)
  - [Members Meeting](#annual-members-meeting)
  - [Board Meeting](#board-mee

**Pull request merged**